qemu/tests/balloon_service.py

```diff
@@ -0,0 +1,183 @@
+import logging
+import time
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import error_context
+from aexpect import ShellCmdError
+
+
+@error_context.context_aware
```

Comment:
Hi coli,
Suggest to update the case name, no idea what will be tested when I read the case name  balloon_service